.github/workflows/ci.yaml

```diff
@@ -107,6 +107,6 @@ jobs:
       lang: go
       # Latest release at the time this was written
       # TODO(bergundy): Find some way to automatically upgrade to "latest"
-      sdk-version: 'v1.16'
+      sdk-version: 'v1.18.1'
```

Comment:
This was required because v1.16.0 no longer works with the latest API repo due to:

> go.temporal.io/api/update/v1: module go.temporal.io/api@latest found (v1.13.0), but does not contain package go.temporal.io/api/update/v1

v1.16.0 and v1.17.0 of the Go SDK are not compatible with latest API module because we had update code in there but have since deleted that API package.